packages/valory/skills/apy_estimation_chained_abci/skill.yaml

```diff
@@ -112,11 +113,11 @@ models:
         alpha: 0.25
         window_size: null
       pair_ids:
-        uniswap_subgraph:
+      - - uniswap_subgraph
         - '0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5'
-        spooky_subgraph:
-        - '0x2a651563c9d3af67ae0388a5c8f89b867038089e'
-        - '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c'
+      - - spooky_subgraph
+        - - '0x2a651563c9d3af67ae0388a5c8f89b867038089e'
+          - '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c'
```

Comment:
@angrybayblade following your recommendation, when defining pair_ids as:
`pair_ids: [["uniswap_subgraph", '0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5'],["spooky_subgraph",['0x2a651563c9d3af67ae0388a5c8f89b867038089e', '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c']]]`

It gets re-formatted into this form when running make hashes.

Reply:
Yeah, it's the yaml format for 2d lists

Reply:
Why does `uniswap_subgraph` has only one  address whereas `spooky_subgraph` has two? @dvilelaf any idea?

Reply:
No, I don't really know